sql/src/test/java/io/crate/execution/engine/collect/stats/JobsLogsTest.java

```diff
@@ -117,6 +117,19 @@ public void testEntriesCanBeRejectedWithFilter() {
         assertThat(Iterables.size(stats.jobsLogSink), is(1));
     }
 
+    @Test
+    public void testFilterIsValidatedOnUpdate() {
+        Settings settings = Settings.builder()
+            .put(JobsLogService.STATS_JOBS_LOG_FILTER.getKey(), "stmt = 'foo'")
+            .build();
+        // creating the service registers the update listener
+        JobsLogService ignored = new JobsLogService(settings, clusterSettings, getFunctions(), scheduler, breakerService);
+
+        expectedException.expectMessage("illegal value can't update [stats.jobs_log_filter] from [true] to [statement = 'x']");
```

Comment:
Why from `[true]` and not from `stmt = 'foo'` ?

Reply:
@matriv The clusterSettings had different settings than those the test defined - I removed the initial setting definition as it's not used.

Reply:
Ok, thx. It was confusing before.